mitodl/mitxpro
# Update styling of enrolled button and add a check mark

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #721 

#### What's this PR do?
Fixes styling of enrolled buttons to look like the mockup. Note that the check mark is slightly different. I used the SVG from open-iconic. Should we look for a narrower check mark?

#### How should this be manually tested?
Go to the program or course run page for a product which you have purchased, and for another product you have not. Verify that they look similar to the screenshots below

#### Screenshots
Enrolled:
![Screenshot from 2019-07-03 14-42-19](https://user-images.githubusercontent.com/863262/60617028-c8297880-9da0-11e9-8ce5-87b08b62b8de.png)

Enroll now (should look the same as before):
![Screenshot from 2019-07-03 14-43-16](https://user-images.githubusercontent.com/863262/60617089-eb542800-9da0-11e9-92d8-becacb05a84b.png)
